docs/java/XML-&-JSON-Data-Formats.md

```diff
@@ -40,7 +40,7 @@ java -jar messagepreview.jar -type json message.json --printAverageStats
 където message.json: 
 
 ```text
```

Comment:
```
could be replaced with `javascript which should format the json properly.

Reply:
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code

Reply:
Changed text to javascript